modules/components/src/AdvancedFacetView/LiveAdvancedFacetView.js

```diff
@@ -81,6 +81,39 @@ const fetchAggregationData = async ({ sqon, extended, projectId, index }) => {
   }));
 };
 
+const removeIds = ({ mapping, extended, parentField = null }) => {
```

Comment:
this seems like a useful function to add to `mapping-utils`. can be out of scope for this, but if you can write a quick issue for that, i'd appreciate it